tests/components/hassio/test_init.py

```diff
@@ -196,15 +196,16 @@ def test_fail_setup_without_environ_var(hass):
 
 
 @asyncio.coroutine
-def test_fail_setup_cannot_connect(hass):
+def test_fail_setup_cannot_connect(hass, caplog):
```

Comment:
test function name need change to reflect current behavior

Reply:
I'll open a standalone PR for that

Reply:
https://github.com/home-assistant/home-assistant/pull/22421